SPECS/glusterfs/glusterfs.spec

```diff
@@ -811,196 +1136,245 @@ exit 0
 fi
 if [ $1 -ge 1 ]; then
-    if [ -f %{glusterfsd_svcfile} ]; then
+    if [ -f %glusterfsd_svcfile ]; then
         %systemd_postun_with_restart glusterfsd
     fi
     %systemd_postun_with_restart glusterd
 fi
 exit 0
-%endif
 
 %preun thin-arbiter
 if [ $1 -eq 0 ]; then
-    if [ -f %{glusterta_svcfile} ]; then
-        %{service_stop} gluster-ta-volume
+    if [ -f %glusterta_svcfile ]; then
+        %service_stop gluster-ta-volume
         %systemd_preun gluster-ta-volume
     fi
 fi
+%endif
 
 ##-----------------------------------------------------------------------------
 ## All %%postun should be placed here and keep them sorted
 ##
 %postun
-%{?ldconfig}
 %if ( 0%{!?_without_syslog:1} )
-
+%if ( 0%{?fedora} ) || ( 0%{?rhel} && 0%{?rhel} >= 6 )
 %systemd_postun_with_restart rsyslog
-
 %endif
-exit 0
+%endif
 
 %if ( 0%{!?_without_server:1} )
 %postun server
-%{?ldconfig}
 %if (0%{?_with_firewalld:1})
     %firewalld_reload
 %endif
 exit 0
 %endif
 
+%if ( 0%{!?_without_server:1} )
+%if ( 0%{?fedora} && 0%{?fedora} > 25  || ( 0%{?rhel} && 0%{?rhel} > 6 ) )
+%postun ganesha
+if [ $1 -eq 0 ]; then
+  # use the value of ganesha_use_fusefs from before glusterfs-ganesha was installed
+  %selinux_unset_booleans ganesha_use_fusefs=1
+fi
+exit 0
+%endif
+%endif
+
+%if ( 0%{!?_without_georeplication:1} )
+%postun geo-replication
+%if ( 0%{?rhel} && 0%{?rhel} >= 8 )
+if [ $1 -eq 0 ]; then
+  %selinux_unset_booleans %{selinuxbooleans}
+fi
+exit 0
+%endif
+%endif
+
+##-----------------------------------------------------------------------------
+## All %%trigger should be placed here and keep them sorted
+##
+%if ( 0%{!?_without_server:1} )
+%if ( 0%{?fedora} && 0%{?fedora} > 25  || ( 0%{?rhel} && 0%{?rhel} > 6 ) )
+# ensure ganesha_use_fusefs is on in case of policy mode switch (eg. mls->targeted)
+%triggerin ganesha -- selinux-policy-targeted
+semanage boolean -m ganesha_use_fusefs --on -S targeted
+exit 0
+%endif
+%endif
+
 ##-----------------------------------------------------------------------------
-## All %%files should be placed here and keep them sorted by groups
+## All %%files should be placed here and keep them grouped
 ##
 %files
-%license COPYING-GPLV2 COPYING-LGPLV3
-%doc ChangeLog INSTALL README.md THANKS COMMITMENT
+%doc ChangeLog COPYING-GPLV2 COPYING-LGPLV3 INSTALL README.md THANKS COMMITMENT
 %{_mandir}/man8/*gluster*.8*
 %if ( 0%{!?_without_server:1} )
 %exclude %{_mandir}/man8/gluster.8*
 %endif
 %dir %{_localstatedir}/log/glusterfs
-%if ( 0%{!?_without_rdma:1} )
-%exclude %{_libdir}/glusterfs/%{version}%{?dev}/rpc-transport/rdma*
-%endif
-%if ( 0%{!?_without_server:1} )
+%if 0%{?!_without_server:1}
 %dir %{_datadir}/glusterfs
 %dir %{_datadir}/glusterfs/scripts
-%{_datadir}/glusterfs/scripts/post-upgrade-script-for-quota.sh
-%{_datadir}/glusterfs/scripts/pre-upgrade-script-for-quota.sh
+     %{_datadir}/glusterfs/scripts/post-upgrade-script-for-quota.sh
+     %{_datadir}/glusterfs/scripts/pre-upgrade-script-for-quota.sh
```

Comment:
Why the indent?

Reply:
upstream indented the lines

Reply:
i'm going to leave this alone unless it is important ... reopen if you feel it is vital.

Reply:
Not vital and at a second glance look actually pretty helpful.